src/core/catchment/giuh/GIUH.cpp

```diff
@@ -100,6 +100,7 @@ void giuh_kernel_impl::set_interpolation_regularity_seconds(unsigned int regular
     }
 }
 
+__attribute__((no_sanitize("address")))
```

Comment:
Per discussion, intent here is to move forward without excising this zombie code.... but this means yes we need to get on with the disposition of this code somehow or another.

Reply:
I opened #587 about this. Initially assigned to you for disposition